### PR TITLE
Reduce eslint config

### PR DESCRIPTION
**Pull request by TrySound**
Thanks for submitting a pull request to RGL!

Please reference an open issue. If one has not been created, please create one along with a failing
example or test case.

Please do not commit built files (`/dist`) to pull requests. They are built only at release.

### CODE REVIEW

**Comment by STRML** [comment]
Will need a rebase on the other PR I just merged

File: lib/components/WidthProvider.jsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ type ProviderT = (
   ComposedComponent: ReactComponentType<any>
 ) => ReactComponentType<any>;
 const WidthProvider: ProviderT = ComposedComponent =>
-  class extends React.Component<Props, State> {
+  class WidthProvided extends React.Component<Props, State> {
```
Comment (STRML) [inline lib/components/WidthProvider.jsx]: Should be `WidthProvider`

Comment (TrySound) [reply lib/components/WidthProvider.jsx]: But there is already WidthProvider.

Comment (STRML) [reply lib/components/WidthProvider.jsx]: Ahhh, I misunderstood the name. Thanks

**Pull request merged**